server/plugin/command_test.go

```diff
@@ -415,6 +415,34 @@ func TestHandleSubscriptions(t *testing.T) {
 	}
 }
 
+func TestHandleCreate(t *testing.T) {
+	p := Plugin{}
+	args := &model.CommandArgs{
+		UserId: testutils.GetID(),
+	}
+	for _, testCase := range []struct {
+		description      string
+		params           []string
+		expectedResponse string
+	}{
+		{
+			description:      "HandleCreate: Invalid number of params",
+			expectedResponse: "Invalid create command. Available commands are 'incident' and 'request'.",
```

Comment:
I think there will be a constant for this and if present use that

Reply:
no there is no constant for this